attributes/attributes.go

```diff
@@ -50,6 +50,9 @@ func New(kvs ...interface{}) *Attributes {
 // times, the last value overwrites all previous values for that key.  To
 // remove an existing key, use a nil value.
 func (a *Attributes) WithValues(kvs ...interface{}) *Attributes {
+	if a == nil {
+		a = New()
```

Comment:
`return New(kvs)`.

Note that `Value` doesn't handle a `nil` `a`.

Reply:
Done for both.